addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -35,15 +35,17 @@ const CountdownWidget = publicWidget.Widget.extend({
         this.progressBarColor = this._ensureCssColor(this.el.dataset.progressBarColor);
 
         this.onlyOneUnit = this.display === 'd';
-        this.width = parseInt(this.size);
-        if (this.layout === 'boxes') {
-            this.width /= 1.75;
-        }
-        this._initTimeDiff();
 
+        this._updateCanvasSize();
```

Comment:
At this time, the parent's width could still change so the size will be wrong. Actually when the countdown starts the editor panel is not ready and it will change the page width when it appears. See below

![FireShot Capture 167 - Capture d’écran de 2020-10-16 16-10-37 png (3840×1080) - ](https://user-images.githubusercontent.com/52911687/96269145-4592ef80-0fca-11eb-8c2c-e429ac3290ad.jpg)

Other bug, when dropping the countdown in a column.. It's hidden

![FireShot Capture 170 - Capture d’écran de 2020-10-16 16-12-49 png (3840×1080) - ](https://user-images.githubusercontent.com/52911687/96269435-a7ebf000-0fca-11eb-979d-6599efe383da.jpg)

Reply:
About bug, It is already exists in the master branch, I have checked at runbot revision (4588664). It's not come from my code.  

![Website_countdown](https://user-images.githubusercontent.com/7599806/96333127-bf3be380-1085-11eb-9818-419d56fc159e.png)